packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -496,12 +496,7 @@ export class InputTimePicker
     syncHiddenFormInput("time", this, input);
   }
 
-  private timeChangeHandler(event): void {
-    if (this.disabled || !event.composedPath().includes(this.el)) {
-      return;
-    }
-
-    const newValue = event.detail;
+  private timeChangeHandler({ detail: newValue }): void {
```

Comment:
@eriklharper do you still need the `disabled` check?

Reply:
Can't hurt for sure :)